tests/src/testFEElement.hh

```diff
@@ -23,7 +23,7 @@
  * the finite element
  */
 template <template <typename> typename FEElementTemplate, int gridDim, typename PreBasis, typename... F>
-auto testFEElement(const PreBasis& preBasis, const std::string& elementName, F&&... f) {
+auto testFEElement(const PreBasis& preBasis, const std::string& elementName, const bool& isRandomGrid, F&&... f) {
```

Comment:
maybe name this boolean something like `onDistortedElement`or something less misleading.